Sources/Core/Other/PasteboardMonitor.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  PasteboardMonitor.swift
```

Comment:
Please make the header comment match header comments in other files in terms of date format and copyright format.